packages/api/src/sdk-bundle-api/sdk-to-bundle/test-run-environment.ts

```diff
@@ -25,8 +25,14 @@ export interface TestRunGitHubPullRequestContext {
   /** Base commit hash */
   baseSha: string;
 
+  /** Base ref (usually /refs/head/<branch>) */
+  baseRef?: string;
+
   /** Head commit hash */
   headSha: string;
+
+  /** Head ref (usually /refs/head/<branch>) */
+  headRef?: string;
 }
```

Comment:
Note we may also want to expose the details required to construct a URL to the test run